docs/source/model_design/vivarium_model_components/risk_factors/risk_exposure/index.rst

```diff
@@ -61,6 +58,35 @@ Risk exposure distributions can be:
 
  - Continuous
 
+See the table for examples of each of these risk exposure distributions. 
+
+.. list-table:: Risk exposure distributions
+  :header-rows: 1
+
+  * - Distribution
+    - Example
+    - Possible exposure values
+  * - Dichotomous
+    - Zinc deficiency
+    - * Not deficient
+      * Deficient
+  * - Unordered polytomous
+    - Unsafe water source
```

Comment:
This exposure is partially ordered---it has two dimensions so some levels are comparable, but others are not.  Not sure if that is too much information to pack in here, but it could potentially reduce confusion to make a note of it.

Reply:
Thanks for noting this. I changed it to tobacco use (none/chewing/cigarettes/cigars) which I think is a better fit. I also added a note about risk exposures being mutually exclusive and collectively exhaustive because it made me think of that.